dotnet/src/Connectors/Connectors.OpenAI/Core/ClientCore.ChatCompletion.cs

```diff
@@ -181,12 +181,12 @@ internal async Task<IReadOnlyList<ChatMessageContent>> GetChatMessageContentsAsy
             }
 
             // If we don't want to attempt to invoke any functions, just return the result.
-            if (!toolCallingConfig.AutoInvoke)
+            if (!(toolCallingConfig?.AutoInvoke ?? false))
             {
                 return [chatMessageContent];
             }
 
-            Debug.Assert(kernel is not null);
+            // Debug.Assert(kernel is not null);
```

Comment:
Is this change intentional?

Reply:
Leaving this was breaking my integration tests (breaking the test runtime)